spec/blueprints/category_blueprint_spec.rb

```diff
@@ -10,7 +10,7 @@
       let(:blueprint_category) { CategoryBlueprint.render_as_hash(category, view: :basic) }
 
       it "validates numbers of elements" do
-        expect(blueprint_category.count).to eql(2)
+        expect(blueprint_category.count).to eql(3)
```

Comment:
strange test just tot test number of keys inside
no need, remove it
better to test exact value it returns
as I noted below